pkg/agent/secondarynetwork/init.go

```diff
@@ -50,6 +51,7 @@ func NewController(
 	podUpdateSubscriber channel.Subscriber,
 	primaryInterfaceStore interfacestore.InterfaceStore,
 	secNetConfig *agentconfig.SecondaryNetworkConfig, ovsdb *ovsdb.OVSDB,
+	nodeConfig *config.NodeConfig,
```

Comment:
Nit: could we move `nodeConfig` to be after `interfaceStore` and before `secNetConfig`?